project/Version.scala

```diff
@@ -21,7 +21,7 @@ object Version {
     Properties.envOrElse(environmentVariable, default)
 
   val geotrellis  = "0.10.0-SNAPSHOT"
-  val scala       = "2.11.5"
```

Comment:
Spark 1.2 introduced support for 2.11 but both my local and cluster spark assembly are still built for 2.10. We will have to figure out what that deployment looks like. We still have the cross build, but maybe let's keep 2.10 as default for now.